docs/CONTRIBUTING.md

```diff
@@ -145,279 +145,12 @@ to check your formatting, you have the following options:
      The document updates automatically
      each time you write your changes to disk.
 
-## Interacting with github
+## [Interacting with github](content/en/contribute/general/github/_index.md)
```

Comment:
We should not hide links in headings, and also do not reference the .md file but just the folder. When this gets rendered it turns into a index.html and there will be no _index.md

Reply:
whoops my bad, the link is correct - but it should not be hidden within the heading

Reply:
ok